### PR TITLE
Define module as using bitcoin style encodings

### PR DESCRIPTION
**Pull request by sublimator**
This updates the broken base2 tests to pass, and adds fair warning what to expect.

### CODE REVIEW

**Comment by dcousens** [comment]
Thanks @sublimator,  is this specific to only bitcoin?  Seems oddly specific.

**Comment by sublimator** [comment]
>>> is this specific to only bitcoin? Seems oddly specific.

I can't honestly say I know :)

File: README.md [hunk 1]
```diff
@@ -35,6 +35,38 @@ var bs62 = require('base-x')(BASE62)
...
+digits = []
+while significant > base:
+  significant, remainder = divmod(significant, base)
+  digits.append(remainder)
+digits.append(significant)
+assert list(reversed(digits)) == [3,0,3,9]
+assert hex(12345) == '0x3039'
+```
+
+Of course the input is actually an array of digits already :)
```
Comment (dcousens) [inline README.md]: This was a really good explanation.  Thanks @sublimator

Comment (sublimator) [inline README.md]: :) Cheers

**Pull request merged**